modules/nw-ovn-kubernetes-change-log-levels.adoc

```diff
@@ -6,7 +6,7 @@
 [id="nw-ovn-kubernetes-change-log-levels_{context}"]
 = Changing the OVN-Kubernetes log levels
 
-The default log level for OVN-Kubernetes is 2. To debug OVN-Kubernetes set the log level to 5.
+The default log level for OVN-Kubernetes is 2. To debug OVN-Kubernetes, set the log level to 5.
```

Comment:
> Additionally, current documentation states that log level is 2, but it is observed to be 4 by default as noted above.

Reply:
The default is `4`

https://github.com/openshift/cluster-network-operator/blob/release-4.14/bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-node.yaml#L855

```
        - name: OVN_KUBE_LOG_LEVEL
          value: "4"
```